BUILDING.md

```diff
@@ -357,16 +365,20 @@ Since CHERI support is available only for a few platforms, verifying CHERI-C
 programs that use header files from the C standard library will require a setup
 of a C standard library for one of these platforms.
 
+To obtain and install a CHERI sysroot, the
+[cheribuild](https://github.com/CTSRD-CHERI/cheribuild)
+tool is the recommended way:
+```
+cheribuild.py cheribsd-mips64-hybrid disk-image-mips64-hybrid
```

Comment:
Include `-d` and switch to the riscv64 versions.

Reply:
done. Thanks

Reply:
Done Thanks!